src/cloud.ts

```diff
@@ -29,6 +41,81 @@ export class Route {
     }
 }
 
+export class Client {
+    base: string
+    token: string
+    _httpclient: httpm.HttpClient
+
+    constructor(token: string) {
+        this.base = DEFAULT_CLOUD_URL
+        this.token = token
+        this._httpclient = new httpm.HttpClient("fermyon/actions", [], {
+            headers: {
+                Authorization: `Bearer ${this.token}`
+            }
+        })
+    }
+
+    async getAllApps(): Promise<App[]> {
+        const resp = await this._httpclient.get(`${this.base}/api/apps`)
+        if (resp.message.statusCode !== httpm.HttpCodes.OK) {
+            throw `expexted code ${httpm.HttpCodes.OK}, got ${resp.message.statusCode}`
+        }
+
+        const appsResp: GetAppsResp = JSON.parse(await resp.readBody())
+        return appsResp.items;
+    }
+
+    async getAppIdByName(name: string): Promise<string> {
+        let apps = await this.getAllApps()
+        const app = apps.find(item => item.name === name);
+        if (!app) {
+            throw `no app found with name ${name}`
+        }
+
+        return app.id;
+    }
+
+    async deleteAppById(id: string): Promise<void> {
+        const resp = await this._httpclient.get(`${this.base}/api/apps`)
+        if (resp.message.statusCode !== httpm.HttpCodes.OK) {
+            throw `expexted code ${httpm.HttpCodes.OK}, got ${resp.message.statusCode}`
```

Comment:
```suggestion
            throw `expected code ${httpm.HttpCodes.OK}, got ${resp.message.statusCode}`
```

Reply:
done :)

Reply:
Oh I think you might not have pushed your latest changes -- can you double-check?

Reply:
yep, forgot to push the code. done now :)